main.go

```diff
@@ -1,17 +1,52 @@
 package main
 
 import (
+	"errors"
 	"fmt"
-	"log"
+	"io"
+	"log/slog"
+	"os"
+	"time"
 
 	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
+	sloggin "github.com/samber/slog-gin"
 )
 
+func init() {
+	err := os.Mkdir("logs", 0777)
```

Comment:
I guess this only works if the dir doesnt exist already?

Reply:
Aye aye captain. It is handled by "!errors.Is(err, os.ErrExist)" below :)